gui.go

```diff
@@ -29,9 +29,18 @@ type Text struct {
 
 // SliderWannabe represents abstract of slider that changes a given parameter
 type SliderWannabe struct {
-	y         float64
-	parameter *Parameter
-	format    string
+	y           float64
+	canvasWidth float64
+	parameter   *Parameter
+	format      string
+}
+
+// SwitchWannabe represents abstract of switch that can switch between various position integrator
+// methods
+type SwitchWannabe struct {
+	y                  float64
+	canvasWidth        float64
```

Comment:
The same. I can guess why we need `canvasWidth`? Explanation or descriptive name is necessary here to understand the context.